### ISSUE TITLE
Problem with Image field

### ISSUE THREAD

**Issue by IsmailAshour** [onkbear/backpack-nested-crud#8]
There is a problem while using the image field in the nested crud.

The first problem is that I can't resize the image.
The second problem is that sometimes upload succeeded and sometimes failed

do you have any idea to fix this?

**Comment by onkbear** [comment]
Hi @drashoor,
It is working in my local environment.
Have you followed the instruction of `image` field below?
https://backpackforlaravel.com/docs/4.0/crud-fields#image

Could you please make sure your image field works outside of `nested crud` field?

Thanks,

**Comment by IsmailAshour** [comment]
Yes, I followed the instructions.
when I try to solve this issue, 
I note that the posted base64 image field is empty when failed.

the second question yes works fine on the same project outside nested crud.

**Comment by onkbear** [comment]
@drashoor, Hmm...
Could you please provide me package versions of `laravel/framework` and `backpack/crud` ?
Which web browser are you using ?
I also would like to see the code how did you add the `image` field via `CRUD::addField`.

**Comment by IsmailAshour** [comment]
laravel/framework                    v7.7.1 
backpack/crud                        4.0.60 

Firefox, Opera, Chrome.

`        
$this->crud->addField([
            'label' => 'Image',
            'name' => 'image',
            'type' => 'image',
            'upload' => true,
            'crop' => true,
            'aspect_ratio' => 1.7,
            'disk' => 'uploads',
        ]);
`

**Comment by onkbear** [comment]
I could have same issue, thank you! I will have a look.

**Comment by ibpavlov** [comment]
I had similar issue before and it turned out that there were some file size limitations. Try with some small file. And is the request randomly failing with the same image or with different?

**Comment by onkbear** [comment]
Hi @drashoor,

When `image` has crop option, it creates an image on the form submit event.
`backpack-nested-crud` cannot handle it therefor, it is happened.
I have changed the way to handle form submit event.
Please `composer update` this package to `1.1.0`.

But, you still need to modify `image.blade.php` field below. Please copy to your own folder.

``` diff
    $mainImage.cropper(options).cropper("reset", true).cropper("replace", this.result);
    // Override form submit to copy canvas to hidden input before submitting
-    $('form').submit(function() {
-        var imageURL = $mainImage.cropper('getCroppedCanvas').toDataURL(file.type);
-        $hiddenImage.val(imageURL);
-        return true; // return false to cancel form action
-    });
+    if (element.closest('.modal-content').length) {
+        element.closest('.modal-content').find('.save-block').click(function(){
+            var imageURL = $mainImage.cropper('getCroppedCanvas').toDataURL(file.type);
+            $hiddenImage.val(imageURL);
+        })
+    } else if (element.closest('form').length) {
+        element.closest('form').submit(function(){
+            var imageURL = $mainImage.cropper('getCroppedCanvas').toDataURL(file.type);
+            $hiddenImage.val(imageURL);
+            return true; // return false to cancel form action
+        });
+    }
    $rotateLeft.click(function() {
        $mainImage.cropper("rotate", 90);
    });
```

I think Backpack 4.1 can be compatible with this package. I may be able to create PR.

There is similar functionality using Ajax.
https://backpackforlaravel.com/docs/4.1/crud-fields#relationship

**Comment by IsmailAshour** [comment]
@onkbear 

works fine. thanks alot.
